db/collection.go

```diff
@@ -95,19 +95,23 @@ func (c *Collection) WriteTxn(f func(txn *Txn) error) error {
 	return c.db.writeTxn(c, f)
 }
 
-// FindByID finds an instance by its ID and saves it in v.
+// FindByID finds an instance by its ID.
 // If doesn't exists returns ErrNotFound.
-func (c *Collection) FindByID(id core.InstanceID, v interface{}) error {
-	return c.ReadTxn(func(txn *Txn) error {
-		return txn.FindByID(id, v)
+func (c *Collection) FindByID(id core.InstanceID) (instance []byte, err error) {
```

Comment:
Direct go api changes here... find by id returns bytes now and below crate returns the list of ids